common/src/main/org/jetbrains/lincheck/descriptors/AccessLocation.kt

```diff
@@ -54,11 +88,14 @@ class AccessPath(val locations: List<AccessLocation>) {
         for (location in locations) {
             when (location) {
                 is LocalVariableAccessLocation -> {
-                    builder.append(location.variableName)
+                    if (!isThisName(location.variableName)) {
```

Comment:
Should we always apply the logic with `if (!isThisName(location.variableName)) { .. }`?

Reply:
> Should we always apply the logic with if (!isThisName(location.variableName)) { .. }?

What do you mean by "always" here? Do you mean "for other branches of this `when`" ?